libbpf-tools/klockstat.c

```diff
@@ -93,6 +94,20 @@ static const char program_doc[] =
 "  klockstat -P                  # print stats per thread\n"
 ;
 
+static const char *internal_ksym_names[] = {
```

Comment:
Comparing the address of each symbol appears to be more cost-effective than comparing the symbol strings.